### DISCUSSION TITLE
Create Model Error

### DISCUSSION THREAD

**Discussion by scalenow** [mindsdb/mindsdb#9487] [Get help]
Folks ,trying to run this command and getting the error..This should be a standard execution as straight lift from website. Any thoughts.Thanks in advance ..................................................................... CREATE MODEL openai_model
PREDICT target_column
USING
      engine = 'openai_engine',  -- engine name as created via CREATE ML_ENGINE
      mode = 'mode_name', -- optional, ...
      model_name = 'openai_model_name',  -- optional with default value of gpt-3.5-turbo
      question_column = 'question',  -- optional, column name that stores user input
      context_column = 'context',  -- optional, column that stores context of the user input
      prompt_template = 'input your query here', -- optional, user provides instructions to the model here
      max_tokens = 100, -- optional, token limit for answer
      temperature = 0.3, -- temp.   Error     SQL statement cannot be parsed by mindsdb_sql - CREATE MODEL openai_model
PREDICT target_column
USING
      engine = 'openai_engine',  -- engine name as created via CREATE ML_ENGINE
      mode = 'mode_name', -- optional, ...
      model_name = 'openai_model_name',  -- optional with default value of gpt-3.5-turbo
      question_column = 'question',  -- optional, column name that stores user input
      context_column = 'context',  -- optional, column that stores context of the user input
      prompt_template = 'input your query here', -- optional, user provides instructions to the model here
      max_tokens = 100, -- optional, token limit for answer
      temperature = 0.3, -- temp: Syntax error, unexpected end of query:
>                                                                  prompt_template = 'input your query here',
>                                                                 max_tokens = 100,
>                                           temperature = 0.3,
--------------------------------------------------------------^
Expected symbol: "[identifier]" (edited)

**Comment by ZoranPandovski** [comment]
Hey @scalenow the above query is a template, so copy/paste will not work. You will need to setup the parameters related to your use case

**Reply by scalenow** [reply]
@ZoranPandovski  Good to see you here ,its Abhi Chaturvedi

Thank you and apologies for my ignorance, isnt all the columns are listed as optional should I make them blank
